src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -66,53 +70,73 @@ public String getPropertyName() {
 	}
 	
 	@Override
-	public Class<Short> getReturnType() {
-		return Short.class;
+	public Class<Object> getReturnType() {
```

Comment:
making an expression return an object isn't as simple as this sadly, take a look at the ternary expression